CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # Bcoin Release Notes & Changelog
 
+## v1.x
+
+### Migration
+
+The chain indexing subsystem has been refactored to be more modular and
+flexible. A migration is required to cleanup the old indexes, if present.
```

Comment:
Need to mention potential downtime due to re-indexing after running migrations to delete old indexes.

Reply:
Done.